docs/releases/3.1.5.txt

```diff
@@ -16,3 +16,8 @@ Bugfixes
 * Fixed a bug in Django 3.1 that caused a crash when processing middlewares in
   an async context with a middleware that raises a ``MiddlewareNotUsed``
   exception (:ticket:`32299`).
+
+* Fixed a regression in Django 3.1 that caused the incorrect prefixing of
+  ``STATIC_URL`` and ``MEDIA_URL`` settings, by the server-provided value of
+  ``SCRIPT_NAME`` (or ``/`` if not set), when set to an absolute URL without a
+  domain, e.g. ``http://myhost/`` (:ticket:`32304`).
```

Comment:
Maybe...

```suggestion
  ``SCRIPT_NAME`` (or ``/`` if not set), when set to a URL specifying the
  protocol but without a top-level domain, e.g. ``http://myhost/``
  (:ticket:`32304`).
```

(but not fussed if you don't like it.)

Reply:
Updated, thanks :+1: